js/src/gui.js

```diff
@@ -262,6 +262,10 @@ class StageWidget{
           // FIXME
           var sw = sidebar.dom.getBoundingClientRect().width
           var ew = el.getBoundingClientRect().width
+          if (ew == sw) {
```

Comment:
what if `ew = 300`? 

Using percentage seems better.

Reply:
swapped it